### ISSUE TITLE
Vulkan error flood on 2d bloom example

### ISSUE THREAD

**Issue by Azorlogh** [bevyengine/bevy#8684]
## Bevy version

Tested with bevy 0.10.1 and also commit bc9144bcd68999fed29840a57c60c63b79ffc531 

## \[Optional\] Relevant system information

OS: Arch linux 6.3.1-zen
Vulkan instance version: 1.3.246

Adapter:
`AdapterInfo { name: "NVIDIA GeForce RTX 2060", vendor: 4318, device: 7817, device_type: DiscreteGpu, driver: "NVIDIA", driver_info: "530.41.03", backend: Vulkan }`

## What you did

Ran the 2d bloom example

## What went wrong

The following error floods the console endlessly:
```
2023-05-25T21:58:32.096948Z ERROR wgpu_hal::vulkan::instance: VALIDATION [VUID-vkCmdEndDebugUtilsLabelEXT-commandBuffer-01912 (0x56146426)]
    Validation Error: [ VUID-vkCmdEndDebugUtilsLabelEXT-commandBuffer-01912 ] Object 0: handle = 0x7f524c2966f0, name = bloom_upsampling_pass, type = VK_OBJECT_TYPE_COMMAND_BUFFER; | MessageID = 0x56146426 | vkCmdEndDebugUtilsLabelEXT() called without a corresponding vkCmdBeginDebugUtilsLabelEXT first The Vulkan spec states: There must be an outstanding vkCmdBeginDebugUtilsLabelEXT command prior to the vkCmdEndDebugUtilsLabelEXT on the queue that commandBuffer is submitted to (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdEndDebugUtilsLabelEXT-commandBuffer-01912)
2023-05-25T21:58:32.096968Z ERROR wgpu_hal::vulkan::instance:     command buffers: bloom_upsampling_pass
2023-05-25T21:58:32.096978Z ERROR wgpu_hal::vulkan::instance:     objects: (type: COMMAND_BUFFER, hndl: 0x7f524c2966f0, name: bloom_upsampling_pass)
```

## Additional information

I cannot notice anything wrong the actual bloom rendering, it visually looks fine.

**Comment by JMS55** [comment]
Can't reproduce
```
AdapterInfo { name: "Intel(R) UHD Graphics 620 (KBL GT2)", vendor: 32902, device: 22807, device_type: IntegratedGpu, driver: "Intel open-source Mesa driver", driver_info: "Mesa 22.3.7", backend: Vulkan }

SystemInfo { os: "Linux 37 Fedora Linux", kernel: "6.1.18-200.fc37.x86_64", cpu: "Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz", core_count: "4", memory: "7.7 GiB" }
```

**Comment by JMS55** [comment]
This might be caused by https://github.com/gfx-rs/wgpu/pull/3809

**Comment by tecbeast42** [comment]
I have the same issue, also on Arch linux 6.3.9 with an AMD gpu. 

Examples:

- bloom_2d
- bloom_3d 

have the same behaviour

```
AdapterInfo { name: "AMD Radeon Graphics", vendor: 4098, device: 5686, device_type: IntegratedGpu, driver: "AMD open-source driver", driver_info: "2023.Q2.3 (LLPC)", backend: Vulkan }

{ os: "Linux rolling Arch Linux", kernel: "6.3.9-arch1-1", cpu: "AMD Ryzen 7 4800H with Radeon Graphics", core_count: "8", memory: "62.2 GiB" }
```

**Comment by tecbeast42** [comment]
Error persist on current dev-main 9309d89bb84c9e9708c0061fb495af104063308b

**Comment by Kanabenki** [comment]
@tecbeast42 It seems the validation layer fix was merged in wgpu 0.17.0. Could you test again on the current main, since it uses 0.17.1, and your test was done when it used 0.16.0 ? I couldn't reproduce both on the current main (8efcbf3e4f1b645b6fefd979fa72e6e47f10dc6a) and on 9309d89bb84c9e9708c0061fb495af104063308b.

**Comment by Kanabenki** [comment]
@alice-i-cecile It seems this can be closed then :slightly_smiling_face:
